pages/_document.js

```diff
@@ -1,4 +1,5 @@
 import { Html, Head, Main, NextScript } from "next/document";
+import React from "react";
```

Comment:
Why has `React` been imported unnecessarily?

Reply:
May be because I was using react plugin. I have updated the plugin & remove this extra imports. Thanks